securedrop/db.py

```diff
@@ -109,7 +108,8 @@ def documents_messages_count(self):
             for submission in self.submissions:
                 if submission.filename.endswith('msg.gpg'):
                     self.docs_msgs_count['messages'] += 1
-                elif submission.filename.endswith('doc.gz.gpg') or submission.filename.endswith('doc.zip.gpg'):
+                elif submission.filename.endswith('doc.gz.gpg') or \
+                        submission.filename.endswith('doc.zip.gpg'):
```

Comment:
I'm not sure if the spacing on this bottom line is right. Further, I think we should follow PEP8 advice here:

> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

I also think think line breaks should come before binary operators (see https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator).